### PR TITLE
get & post

### PR DESCRIPTION
**Pull request by Jaivardhan1**
Need to make changes to post

### CODE REVIEW

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/post.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import { baseURL } from '../../Constants/Config'
+// Universal fetch request using axios
+export default function universalFetch(setResponse, endpoint, onError, onSuccess) {
+    console.log('Fetch started');
```
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/post.js:5]: We need extra parameter of "requestBody" for POST request.

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/post.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    setResponse({
+        data: null,
+        loading: true,
+        error: null,
+    });
+    axios
+        .post(baseURL + endpoint)
+        .then((resp) => {
```
Comment (Enoch-Hsiao) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/post.js:13]: @HerlinRijo POST requests take in 2 parameters right?

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/post.js (hunk 1 above)
Comment (Jaivardhan1) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/post.js:5]: done

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/post.js (hunk 2 above)
Comment (Jaivardhan1) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/post.js:13]: changed

**Review by Enoch-Hsiao — APPROVED** [review]
I think this is correct.

**Pull request merged**